AmplifyTools/AmplifyXcode/Sources/AmplifyXcode/CLICommandImportConfig.swift

```diff
@@ -10,14 +10,14 @@ import ArgumentParser
 import AmplifyXcodeCore
 
 /// CLI command invoking `CommandImportConfig`.
-struct CLICommandImportConfig: ParsableCommand, CommandExecutable, CLICommandReportable, CLICommandEncodable {
-    static var paramsRegistry = CLICommandEncodableParametersRegistry()
+struct CLICommandImportConfig: ParsableCommand, CommandExecutable, CLICommandReportable, CLICommand {
+    static var parameters: Set<CLICommandParameter> = []
```

Comment:
Style nit: We've been preferring to let type inference do its thing:
```suggestion
    static var parameters = Set<CLICommandParameter>()
```